go.mod

```diff
@@ -13,9 +13,15 @@ require (
 	github.com/openshift/library-go v0.0.0-20200326104512-4dc124c2ddb8
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.4.0
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
+	gopkg.in/gcfg.v1 v1.2.0
+	k8s.io/api v0.18.0
 	k8s.io/apimachinery v0.18.0
 	k8s.io/client-go v0.18.0-beta.2
 	k8s.io/component-base v0.18.0-beta.2
 	k8s.io/klog v1.0.0
+	k8s.io/legacy-cloud-providers v0.18.0-beta.2
```

Comment:
The reason i have decided to vendor in the module is because it allows us to keep bumping the API used for cloud-config otherwise we would be stuck carry a copy..

Reply:
> The reason i have decided to vendor in the module is because it allows us to keep bumping the API used for cloud-config otherwise we would be stuck carry a copy..

Just that one struct? We'd rather have the copy.  These are legacy for a reason and we want to be in control of our API surface.

Reply:
done, copied the config type in `/kube_cloud_config/internal/aws`